vcrBuildLocal.sh

```diff
@@ -0,0 +1,5 @@
+ #!/bin/bash
```

Comment:
A comment here would be nice. Indicating that this is to build dependencies and build the app in the machine used by VCR rather than uploading the build built locally https://developer.vonage.com/en/vonage-cloud-runtime/guides/manifest#build-script